kernel/indtypes.mli

```diff
@@ -33,4 +33,4 @@ type inductive_error =
 [@@ocaml.deprecated "Use [Type_errors.inductive_error]"]
 
 exception InductiveError of Type_errors.inductive_error
-[@@ocaml.deprecated "Use [Type_errors.InductiveError]"]
+(* [@@ocaml.deprecated "Use [Type_errors.InductiveError]"] *)
```

Comment:
??

Reply:
This is actually an error now, the attribute was a no-op but < 4.08 didn't complain.

Reply:
>the attribute was a no-op but < 4.08 didn't complain.

I just tested and I do get a deprecated warning if I put `raise (Indtypes.InductiveError (assert false))` somewhere, and no warning if I remove the attribute. This is on 4.07.1.

Reply:
Maybe a 4.08 bug?

Reply:
IIUC https://github.com/ocaml/ocaml/pull/1705/ changed @@ on exceptions to handle more stuff, and the correct way to deprecate an exception is just 1 @ (like with constructors). Just 1 @ works for me at least.
We can either go with 1 @ or drop the re-export entirely.